<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="productDisplay">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>811</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="updateLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>620</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Последне обновление: никогда</string>
    </property>
   </widget>
   <widget class="QToolButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>620</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon-refresh-128.png</normaloff>icon-refresh-128.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="catalogTreeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Каталог продукции</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продуктов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>271</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Категорий:</string>
    </property>
   </widget>
   <widget class="QLabel" name="categoryCounter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>620</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productCounter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>640</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productThumbLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Нет изображения</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productName">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Название товара</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="productPrice">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>421</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Цена:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
